src/components/Transactions.tsx

```diff
@@ -16,6 +11,7 @@ import ScrollPane from './ScrollPane'
 import Utils from '../utils/utils'
 import AddressBook from './Addressbook'
 import routes from '../constants/routes.json'
+import { PastelURITarget } from '../utils/uris'
```

Comment:
This is also not safe. I understand that this import was missing, but fixing bugs in legacy code can be punishing. Is it possible not to import this and not trigger a ts error?